packages/libraries/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-native",
-  "version": "0.33.7",
+  "version": "0.33.8",
```

Comment:
NPM says last published is 0.33.7

https://www.npmjs.com/package/@fluentui/react-native